apps/backoffice-v2/src/pages/Entity/hooks/useTasks/useTasks.tsx

```diff
@@ -81,7 +81,7 @@ export const useTasks = ({
               {
                 id: 'nested-details-heading',
                 type: 'heading',
-                value: 'Registry information',
+                value: `Plugin (${key})`,
```

Comment:
@Omri-Levy Its a change i had to make so it wont be hard coded on the "Registry information" plugin